streamly-examples.cabal

```diff
@@ -208,8 +208,8 @@ executable CirclingSquare
   if flag(sdl2)
     buildable: True
     build-Depends:
-          text >= 1.2.3 && < 1.2.5
-        , sdl2  >= 2.5.0 && < 2.6
+          text >= 1.2.3 && < 1.3
```

Comment:
Doesn't it work with text-2.0?

Reply:
`sdl2 < 2.6` requires `text < 1.3`

Reply:
But then it will automatically be constrained by sdl2, why should we constrain it.